Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:fermium-buster
+FROM node:fermium-buster AS base
 
 WORKDIR /instant
```

Comment:
Perhaps we could name this something more generic to allow more logical reuse?
`/packages` ?
`/components` ?

Reply:
Agreed, but that might break packages that expect config to be in the instant volume. We'd have to re-look at how the volume is mounted before we make changes. I'm actually thinking that we need to do some thinking and re-work of the how the instant volume works. I think we need to ensure the latest from the image is always loaded in there. I think we should look at this as a separate piece of work.